docs/data/material/pages.ts

```diff
@@ -121,6 +121,10 @@ const pages: MuiPage[] = [
             title: 'Click-Away Listener',
           },
           { pathname: '/material-ui/react-css-baseline', title: 'CSS Baseline' },
+          {
+            pathname: '/material-ui/react-init-color-scheme-script',
+            title: 'Init Color Scheme Script',
```

Comment:
Maybe we should keep it as a CamelCaseString to match how we style it elsewhere?

```suggestion
            title: 'InitColorSchemeScript',
```